src/components/AddBook.css

```diff
@@ -1,36 +1,40 @@
-.form-container {
-  width: 80%;
+.FormDiv {
+  width: 67.9%;
   height: 2.625rem;
-  margin: 2.313rem 6.25rem 6.25rem;
   padding: 2rem 6.188rem 0 1.688rem;
   border-radius: 4px;
-  background-color: #fff;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 2rem;
+  border-top: 0.1rem solid #c7c7c7;
 }
```

Comment:
- [ ] Kindly make sure your **formDiv** sticks as much as possible to the **[template design](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378)**(e.g.: spacing between & around elements, text_colors, colors, alignments,...)

### Template design
![Screenshot from 2022-11-11 18-16-42](https://user-images.githubusercontent.com/83042742/201385409-db51029a-07b5-4ada-9c2f-95eeb3b31d41.png)

### Your design
![Screenshot from 2022-11-11 18-17-18](https://user-images.githubusercontent.com/83042742/201385542-297ed17f-88ae-43a7-9313-2018653202b8.png)